src/pages/dashboard/index.vue

```diff
@@ -1,354 +1,7 @@
-      <NFTPortfolioMainView
-        key="dashboard"
-        ref="portfolioMainView"
-        :portfolio-wallet="getAddress"
-        :portfolio-tab="currentTab"
-        :portfolio-items="currentNFTClassList"
-        :portfolio-items-show-count="currentNFTClassListShowCount"
-        :portfolio-items-sorting="currentNFTClassListSorting"
-        :portfolio-items-sorting-order="currentNFTClassListSortingOrder"
-        :portfolio-items-sorting-option-list="currentNFTClassSortingOptionList"
-        :portfolio-items-creator-filtering="nftCreatorFilter"
-        :portfolio-items-type-filtering="nftTypeFilter"
-        :portfolio-items-type-filtering-options="nftTypeFilteringOptions"
-        :portfolio-collected-creator-list="nftCreatorInfoListOfCollected"
-        :is-loading-portfolio-items="isLoading"
-        :nft-keyword-list="nftKeywordList"
-        :nft-keyword-filtering="nftKeywordsFilter"
-        @portfolio-change-tab="handleTabChange"
-        @portfolio-change-sorting="handleNFTClassListSortingChange"
-        @portfolio-change-creator="handleNFTClassListCreatorChange"
-        @portfolio-change-type="handleNFTClassListTypeChange"
-        @portfolio-change-keywords="handleNFTKeywordsChange"
-        @infinite-scroll="handleInfiniteScroll"
-        @portfolio-reset-filter="handleClearFilter"
-        @portfolio-input-filter-change-creator="handleCreatorInputFilterChange"
-        @portfolio-input-filter-change-keyword="handleKeywordInputFilterChange"
-      >
-        <template #tab-bar-prepend>
-          <ButtonV2
-            preset="tertiary"
-            size="small"
-            :text="$t('dashboard_portfolio_button')"
-            :class="[
-              'block',
-
-              'laptop:absolute',
-              'laptop:left-0',
-              'laptop:ml-[32px]',
-
-              'rounded-full',
-            ]"
-            @click="goMyPortfolio"
-          >
-            <template #prepend>
-              <IconView />
-            </template>
-          </ButtonV2>
-        </template>
-      </NFTPortfolioMainView>
-
-      <FollowerDialog
-        :is-open-followers-dialog="isOpenFollowersDialog"
-        :wallet-is-fetching-followers="walletIsFetchingFollowers"
-        :populated-followers="populatedFollowers"
-        @close="isOpenFollowersDialog = false"
-        @on-export-followers="handleClickExportFollowerList"
-      />
-      <UserStatsIncomeDetailsDialog
-        :address="getAddress"
-        :is-open-dialog="isOpenIncomeDetailsDialog"
-        :is-loading="isIncomeDetailsLoading"
-        :total-sales="walletTotalSales"
-        :total-royalty="walletTotalRoyalty"
-        :total-resales="walletTotalResales"
-        :sales-details="walletSalesDetails"
-        :royalty-details="walletRoyaltyDetails"
-        :resales-details="walletResalesDetails"
-        :target-type="targetType"
-        @close="isOpenIncomeDetailsDialog = false"
-      />
-    </AuthRequiredView>
-  </Page>
-</template>
```

Comment:
Some times not having `<template>` in a redirect page causes weirded renderer issue, best to keep a placeholder/loading `<template>` block